evm/computation.py

```diff
@@ -7,6 +7,14 @@
     abstractmethod
 )
 
+from typing import (  # noqa: F401
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically